src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+
+.card {
+  display: flexbox;
```

Comment:
`display: flexbox;`? is that a new display property or I missed something? 😄

Reply:
I'm somewhat an innovator myself 
hahaha
thanks and sorry